src/main/scala/ba/sake/hepek/bootstrap3/component/BootstrapGridComponents.scala

```diff
@@ -8,45 +8,17 @@ object BootstrapGridComponents extends BootstrapGridComponents
 trait BootstrapGridComponents extends GridComponents {
   import GridComponents._
 
-  override def row(content: Frag*) =
+  override def mkRow(content: Frag*): Frag =
```

Comment:
why don't you provide a more self explaining signature here? I mean `makeRow`

Reply:
These kinda resemble Scala's `mkString` method on `Seq`s... I'll keep it for now.  
They aren't meant to be used by end-users anyway. Same goes for `ratio2BS` etc.

I'll make them `private[hepek]` so they can be used only internally.